app/src/main/java/ru/androidacademy/gooseeye/data/JsonLoad.kt

```diff
@@ -0,0 +1,109 @@
+package ru.androidacademy.gooseeye.data
+
+import android.content.Context
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+
+
+private val jsonFormat = Json { ignoreUnknownKeys = true }
+
+@Serializable
+private class JsonGenre(val id: Int, val name: String)
+
+@Serializable
+private class JsonActor(
+    val id: Int,
+    val name: String,
+    @SerialName("profile_path")
+    val profilePicture: String
+)
+
+@Serializable
+private class JsonMovie(
+    val id: Int,
+    val title: String,
+    @SerialName("poster_path")
+    val posterPicture: String,
+    @SerialName("backdrop_path")
+    val backdropPicture: String,
+    val runtime: Int,
+    @SerialName("genre_ids")
+    val genreIds: List<Int>,
+    val actors: List<Int>,
+    @SerialName("vote_average")
+    val ratings: Float,
+    @SerialName("vote_count")
+    val votesCount: Int,
+    val overview: String,
+    val adult: Boolean
+)
+
+private suspend fun loadGenres(context: Context): List<Genre> = withContext(Dispatchers.IO) {
+    val data = readAssetFileToString(context, "genres.json")
+    parseGenres(data)
+}
+
+internal fun parseGenres(data: String): List<Genre> {
+    val jsonGenres = jsonFormat.decodeFromString<List<JsonGenre>>(data)
+    return jsonGenres.map { Genre(id = it.id, name = it.name) }
+}
+
+private fun readAssetFileToString(context: Context, fileName: String): String {
+    val stream = context.assets.open(fileName)
+    return stream.bufferedReader().readText()
+}
+
+private suspend fun loadActors(context: Context): List<Actor> = withContext(Dispatchers.IO) {
+    val data = readAssetFileToString(context, "people.json")
+    parseActors(data)
+}
+
+internal fun parseActors(data: String): List<Actor> {
+    val jsonActors = jsonFormat.decodeFromString<List<JsonActor>>(data)
+    return jsonActors.map { Actor(id = it.id, name = it.name, picture = it.profilePicture) }
+}
+
+@Suppress("unused")
```

Comment:
you don't need this annotation

Reply:
Could you please check if you really need this annotation? I'm afraid it is just an outdated line from your copypasted functions. Thanks a lot!